node/resolvers/directives/helper.ts

```diff
@@ -44,6 +55,11 @@ export const validateAdminToken = async (
     }
   }
 
+  console.log({ hasValidAdminToken, orgPermission, tokenType })
+  if (hasValidAdminToken && orgPermission && tokenType === 'user') {
+    await lm.checkUserAdminPermission(account, userEmail, orgPermission)
+  }
```

Comment:
I think we can move this check to inside the try above, right after the `lm.getUserAdminPermissions` call

Reply:
the problem with passing this to the try catch above is that the catch does not block the call, it just logs the error